src/blogposts/makie-showcases/post.md

```diff
@@ -13,6 +13,9 @@
 
 ## [Multibody.jl](https://help.juliahub.com/multibody/dev/#Multibody)
 
+Multibody.jl is a library of model components for ModelingToolkit, an equation-based modeling language. With Multibody.jl, the user can assemble dynamical models for simulation of mechanical systems, such as robots, cars, drive lines and aircraft, by connecting ready-made components such as rods, boxes, joints and wheels. 
+The modeling-language is responsible for deriving the equations of motion of the assembled model, lifting this burden off the shoulders of the modeler. 3D mechanical modeling benefits greatly from being able to visualize the mechanical system, both during the model-building process and while analyzing the result of a simulation. The interactive features of Makie fits this bill perfectly, allowing the user to produce both animations and interactive renders of their model.
+
 ![](https://help.juliahub.com/multibody/dev/examples/robot.gif)
```

Comment:
You may want to grab these, in case the link stops working some time in the future?

Reply:
They are grabbed as part of the build... But it's not perfect right now, since they get grabbed every build. I'll try to make sure it only gets downloaded once and checked into the repo